src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -128,13 +131,15 @@ $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
         obj.layers.set(layer.threejsLayer);
     };
     if (urlSuffix) {
-        const url = metadata.urlPrefix + urlSuffix;
+        let url = metadata.urlPrefix + urlSuffix;
```

Comment:
Yes because when i used lopocs, i had an absolute url. But yes, i forgot to add comment, my bad.